File: src/main/java/com/baulsupp/oksocial/services/dropbox/DropboxUtil.java
package com.baulsupp.oksocial.services.dropbox;

import com.google.common.collect.Sets;
import java.util.Collections;
import java.util.Set;
import okhttp3.Request;

public class DropboxUtil {
  private DropboxUtil() {
  }

  public static final Set<String> API_HOSTS =
      Collections.unmodifiableSet(Sets.newHashSet("api.dropboxapi.com", "content.dropboxapi.com"));

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.dropboxapi.com" + s).build();
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/dropbox/DropboxUtil.kt
package com.baulsupp.oksocial.services.dropbox

import okhttp3.Request

object DropboxUtil {

  val API_HOSTS = setOf("api.dropboxapi.com", "content.dropboxapi.com")

  fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
    return requestBuilder.url("https://api.dropboxapi.com" + s).build()
  }
}
